src/components/flat-table/flat-table-row/flat-table-row.style.ts

```diff
@@ -312,10 +307,9 @@ const StyledFlatTableRow = styled.tr<StyledFlatTableRowProps>`
             css`
               position: -webkit-sticky;
               :after {
-                content: none;
                 border: none;
                 content: "";
-                height: ${getFocusHeight(size)} ${newFocusStyling};
+                height: ${rowHeight}px ${newFocusStyling};
```

Comment:
shouldn't this be
```suggestion
height: ${rowHeight};
${newFocusStyling}
```
]

Reply:
It needs the `px` on the height otherwise it applies it as a percentage. The moving of `${newFocusStyling}` out of the height works though. I hadn't moved this as I figured it had to be done that way for a reason. Guess it was just a mistake originally that still worked 🤷🏻

Reply:
Previously it was a function call that added the `;` as part of the string it returned, now it's the actual pixel value so will need adding